README.md

```diff
@@ -1 +1,52 @@
-# react-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# React Capstone
+
+## Built With
+
+- Html
+- Javascript
+- React
+- Redux
+- React-Bootstrap
+
+## Live Demo
+
+[Covid World Statistic -Netlify](https://covid-world-statistic.netlify.app/)
+
+
+## Setup
+- Clone this project
+```
+- $ git clone `$ https://github.com/ahmetbozaci/react-capstone.git`
+- $ cd react-capstone
+```
+- Install npm packages
+```
+- $ npm install
+```
+- Open website
+```
+- $ npm start
+```
+## Authors
+
+👤 **Ahmet Bozacı**
+- Github:[ahmetbozaci](https://github.com/ahmetbozaci)
+- Twitter:[ahmtbozaci](https://twitter.com/ahmtbozaci)
+- LinkedIn:[ahmetbozaci](https://www.linkedin.com/in/ahmetbozaci/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+https://creativecommons.org/licenses/by-nc/4.0/
```

Comment:
As required, let's give appropriate credit to the author [Nelson Sakwa on Behance.](https://www.behance.net/sakwadesignstudio) 👍